src/js/components/Calendar/__tests__/__snapshots__/Calendar-test.js.snap

```diff
@@ -136,7 +136,7 @@ exports[`Calendar date renders 1`] = `
   -ms-flex-pack: center;
   justify-content: center;
   margin: 0px;
-  padding: 12px;
+  padding: 0px;
```

Comment:
Did you verify that Calendar looks OK, both desktop and mobile?

Reply:
yes I did.